Bus Scheduler/src/shared/components/atoms/image/index.tsx

```diff
@@ -0,0 +1,14 @@
+type ImageType = {
```

Comment:
can we create folders like utilityComponents & formComponents rather than atoms ?, will it be more meaningful ?

Reply:
Keeping as it is for now, as we discussed